docs/workflows/create_sync_publish.rst

```diff
@@ -21,7 +21,7 @@ Create a repository ``foo``
 Create a new remote ``bar``
 ---------------------------
 
-``$ http POST http://localhost:8000/pulp/api/v3/remotes/rpm/ name='bar' url='https://repos.fedorapeople.org/pulp/pulp/fixtures/rpm-unsigned/'``
+``$ http POST http://localhost:8000/pulp/api/v3/remotes/rpm/ name='bar' url='https://repos.fedorapeople.org/pulp/pulp/fixtures/rpm-unsigned/' policy='cache_only'``
```

Comment:
I hesitate to have this set (for pulp_file too) until we have the streamer working, but I wanted to keep this PR basically identical to the pulp_file one.

Reply:
s/cache_only/streamed/

Reply:
Maybe we do want it to be the default for pulp_rpm. That would be a good checkin-question for our call. I'm +1 to it.